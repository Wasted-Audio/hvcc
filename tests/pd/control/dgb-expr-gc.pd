#N canvas 2233 -296 607 356 12;
#X obj 240 16 loadbang;
#X obj 264 280 print;
#X msg 458 160 5 4 7;
#X msg 532 160 5;
#X msg 502 160 5 4;
#X obj 458 137 t b b b;
#X obj 264 254 expr ($f1 + $i3)/$f2;
#X msg 381 160 bang 13 3;
#X obj 279 137 t b b b;
#X msg 279 161 4;
#X msg 308 161 7;
#X msg 338 161 13;
#X obj 176 138 t b b b;
#X msg 205 162 junk;
#X msg 176 162 12;
#X msg 243 162 4;
#X msg 28 163 7 3 junk;
#X msg 97 162 junk 14 9;
#X obj 240 37 t b b b b b b;
#X connect 0 0 18 0;
#X connect 2 0 6 0;
#X connect 3 0 6 0;
#X connect 4 0 6 0;
#X connect 5 0 2 0;
#X connect 5 1 4 0;
#X connect 5 2 3 0;
#X connect 6 0 1 0;
#X connect 7 0 6 0;
#X connect 8 0 9 0;
#X connect 8 1 10 0;
#X connect 8 2 11 0;
#X connect 9 0 6 0;
#X connect 10 0 6 1;
#X connect 11 0 6 2;
#X connect 12 0 14 0;
#X connect 12 1 13 0;
#X connect 12 2 15 0;
#X connect 13 0 6 1;
#X connect 14 0 6 0;
#X connect 15 0 6 2;
#X connect 16 0 6 0;
#X connect 17 0 6 0;
#X connect 18 0 16 0;
#X connect 18 1 17 0;
#X connect 18 2 12 0;
#X connect 18 3 8 0;
#X connect 18 4 7 0;
#X connect 18 5 5 0;
