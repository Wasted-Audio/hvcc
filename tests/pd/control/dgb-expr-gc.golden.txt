[@ 0.000]
print: 3
print: 2.25
print: 3
print: 3
print: 2.42857
inlet: expected 'float' but got 'junk'
print: 2.28571
expr: no method for 'junk'
expr: type mismatch
print: 3.66667
